index.html

```diff
@@ -182,6 +182,254 @@
         </section>
       </section>
     </section>
+    <section class="informative">
+      <h2>
+        Examples of usage
+      </h2>
+      <p>
+        In order to use the API, the developer needs to provide and keep track
+        of an number of key pieces of information. Each of these bits of
+        information are passed to the <a>PaymentRequest</a> constructor as
+        arguments, and subsequently used to update the payment request being
+        displayed to the user. Namely, these bits of information are:
+      </p>
+      <ul>
+        <li>The <var>methodData</var>: a sequence of <a>PaymentMethodData</a>s
+        that represents the <a>payment methods</a> that the site supports
+        (e.g., "we support card-based payments, but only Visa and MasterCard
+        credit cards.").
+        </li>
+        <li>The <var>details</var>: The details of the transaction, as a
+        <a>PaymentDetailsInit</a> dictionary. This includes total cost, and
+        optionally a list of goods or services being purchased, for physical
+        goods, any shipping options. Additionally, it can optionally include
+        "modifiers" to how payments are made. For example, "if you pay with a
+        credit card of type X, it incurs a US$3.00 processing fee".
+        </li>
+        <li>The <var>options</var>: Optionally, a list of things that the site
+        needs to deliver the good or service (e.g., for physical goods, the
+        merchant will typically need an physical address to ship to. For
+        digital goods, an email will usually suffice).
+        </li>
+      </ul>
+      <p>
+        Once a <a>PaymentRequest</a> is constructed, it's presented to the end
+        user via the <a>show()</a> method. The <a>show()</a> returns a promise
+        that, once the user confirms request for payment, results in a
```

Comment:
Will investigate separately. I might have broken something in latest ReSpec release I did today.